script/bump

```diff
@@ -1,4 +1,11 @@
-#!/bin/bash -ex
+#!/bin/bash -e
+
+set -e
```

Comment:
`#!/bin/bash -e` and `set -e`?

Reply:
If you run `bash script/bump`, it won't respect that hashbang in my experience. I'll just use one. :)

Reply:
Ahhhh, yes, that makes perfect sense!